streams/shared/src/main/scala/zio/stream/ZChannel.scala

```diff
@@ -933,6 +933,88 @@ sealed trait ZChannel[-Env, -InErr, -InElem, -InDone, +OutErr, +OutElem, +OutDon
         ZIO.suspendSucceed(interpret(exec.run().asInstanceOf[ChannelState[Env, OutErr]]))
       }
 
+  def toZPipeline[In, Out](implicit
+    In: Chunk[In] <:< InElem,
+    Out: OutElem <:< Chunk[Out],
+    InDone: Any <:< InDone,
+    trace: ZTraceElement
+  ): ZPipeline[Env, OutErr, In, Out] =
+    new ZPipeline[Env, OutErr, In, Out](
+//      transform[Nothing, Chunk[In], Any, OutErr, Chunk[Out], Any](
+//        Predef.identity,
+//        In,
+//        InDone,
+//        Predef.identity,
+//        Out,
+//        Predef.identity
+//      )
+      self.asInstanceOf[ZChannel[Env, Nothing, Chunk[In], Any, OutErr, Chunk[Out], Any]]
```

Comment:
I cheat by using `asInstanceOf`, but get a faster code, because the transformation is only busywork to make type checker happy. What do you think about it?

Reply:
I think it is fine to use `asInstanceOf` here.